seagence/README.md

```diff
@@ -2,16 +2,18 @@
 
 ## Overview
 
-[Seagence][1] is a defect monitoring platform for Java applications. Using ExecutionPath technology (aka GPS Tracker for transactions), Seagence detects unknown defects caused by various issues like swallowed exceptions, multithreading issues, and others, including defects that are disguised in a 200 success HTTP response code.
+[Seagence][1] is a production-grade Realtime Defect Detection and Resolution tool for Java applications. Using ExecutionPath technology, Seagence detects known and unknown defects caused by various issues like multithreading issues, swallowed, handled, and unhandled exceptions, and others, including defects that are disguised in a 200 success HTTP response code.
 
-With this integration, Seagence continuously analyzes the data stream from the Seagence agent to detect defects when they occur, including the root cause of the defect. When a defect is detected, the integration will send an event to Datadog to alert your team. Using the out-of-the-box dashboard, you have visibility into the detected defects and root causes to eliminate debugging and troubleshooting. More details can be found on [SeagenceWeb][2].
+With this integration, Seagence backend continuously analyzes the data stream from the Seagence agent to detect defects when they occur, including the root cause of the defect. When a defect is detected, the integration will send an event to Datadog to alert your team. Using the out-of-the-box dashboard, you have visibility into the detected defects and root causes to eliminate debugging and troubleshooting. More details about the defect can be found on [SeagenceWeb][2].
 
 ## Setup
 
 ### Installation
 
 Visit [Seagence][1] to sign up for free. Once registered, navigate to the Seagence tile on the [Datadog Integrations page][5] and click **Install Integration**. Click **Connect Accounts** on the tile, which guides you through the Datadog OAuth2 flow to grant Seagence access to post Events to your Datadog account.
 
+After connecting accounts, go to "Assets" tab on the tile > Recommended Monitors > Click on "Seagence Defect Detection Monitor". This will redirect you to create the out-of-the-box monitor. Click "Create" at the bottom of the page to install the Seagence monitor.
```

Comment:
```suggestion
After connecting your accounts, go to the "Assets" tab on the tile. Click **Recommended Monitors** > **Seagence Defect Detection Monitor**. This will redirect you to create the out-of-the-box monitor. Click **Create** at the bottom of the page to install the Seagence monitor.
```